src/bp/ui-studio/src/web/views/OneFlow/sidePanel/Library/index.tsx

```diff
@@ -264,7 +265,7 @@ const Library: FC<Props> = props => {
               />
             )}
 
-            {item.type === 'variableType' && (
+            {props.canAdd && item.type === 'variableType' && (
               <Fragment>
                 <Button
                   minimal
```

Comment:
The library is language-agnostic, you could still be able to create them in any language (same for creating new workflows/topics)